pkg/apis/networking/util/helpers.go

```diff
@@ -16,14 +14,15 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package junit
+package util
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
-import (
-	"github.com/onsi/ginkgo/v2"
-)
+// IsDefaultIngressClassAnnotation represents an IngressClass annotation that
+// marks a class as the default IngressClass
+const IsDefaultIngressClassAnnotation = "ingressclass.kubernetes.io/is-default-class"
```

Comment:
Is there an issue with import issues to require defining this in multiple packages. This is defined in `api/networking/v1/well_known_annotations.go`. 

If we have to define the const here, it probably also makes sense to name this:

defaultIngressClassAnnotation

I think we should keep this package private to avoid people importing the symbol from here.